src/components/fabSpeedDial/demoModal/style.scss

```diff
@@ -0,0 +1,16 @@
+.md-fab.md-mini {
+  background-color: #aaa !important;
```

Comment:
Is `!important` necessary (here and below) ?
Couldn't we get away with some more specific selector ?